lib/Doctrine/Migrations/Version.php

```diff
@@ -174,9 +179,18 @@ public function writeSqlFile(
             ->write($path, $direction, $sqlQueries);
     }
 
-    public function getMigration() : AbstractMigration
-    {
-        return $this->migration;
+    public function exec(
```

Comment:
No abbreviations :sob:, anything wrong with `execute()`?

Reply:
I maintained backwards compatability. The `execute()` method already exists and I didn't want to change the signature of that method.